docs/user/environment-variables.rst

```diff
@@ -4,11 +4,11 @@ Understanding environment variables
 ===================================
 
 Read the Docs allows you to define your own environment variables to be used in the build process.
-It also defines a set of :doc:`default environment variables </reference/environment-variables>` with information about your build.
+It also defines a set of :doc:`pre-defined environment variables </reference/environment-variables>` with information about your build.
```

Comment:
There might be more of these, I haven't checked. This is beyond what I can remember, but at some point we started saying "pre-defined environment variables", I'm guessing that it's less ambiguous and much easier to understand when put up next to "user-defined environment variables".